scripts/resume.js

```diff
@@ -0,0 +1,86 @@
+const getResume = filePath => {
+  fetch(filePath)
+    .then(response => {
+      return response.json();
+    })
+    .then(data => {
+      // console.log(data.basics);
+      renderResume(data);
+    });
+};
+
+const renderResume = data => {
+  renderHeader(data.basics);
+  renderContactInfo(data.basics);
+  renderAbout(data.basics);
+  renderProfiles(data.basics.profiles);
+};
+
+///////////////////////////////////////////////////
+//     Resume Section Templates
+///////////////////////////////////////////////////
+// Each function below is a template for a single section
+// of the resume
+
+// The header, containing full name and job title
+const renderHeader = basics => {
+  const html = `
+    <h1>${basics.name}</h1>
+    <h2>${basics.label}</h2>
+  `;
+  $(".header").html(html);
+};
+
+const renderContactInfo = basics => {
+  const html = `
+    <aside>Contact</aside>
+    <div>
+      <p><strong>Email</strong>: ${basics.email}</p>
```

Comment:
as you flesh out more of the contact info, each contact field will also have a lot of similarities with one another (i can see the p tags with strong elements for each label)

I think it might make sense to use a header tag semantically to denote each contact field instead of using a strong tag within a paragraph tag